.github/pull_request_template.md

```diff
@@ -0,0 +1,52 @@
+<!-- Thanks for submitting this pull request for KeY -->
+<!-- Since the project has a strict review policy, please make the -->
+<!-- reviewer's job easier by providing the necessary information -->
+<!-- in the text below. The comments may remain since they will be -->
+<!-- invisible when showing the PR. -->
+
+## Related Issue
+
+<!-- Please remove if this PR is not related to an issue. -->
+<!-- Please add number if it is in answer to an issue. -->
+This pull request answers to issue #.
+
+## Intended Change
+
+<!-- Please give a brief description of what behaviour changes and 
+     why it should be changed. -->
+
+## Type of pull request
+
+<!--- What types of changes does your code introduce? Put an `x` in the box(es) that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Refactoring (behaviour should not change or only minimally change)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+- [ ] There are changes to the (Java) code
+- [ ] There are changes to the taclet rule base
+- [ ] There are changes to the deployment/CI infrastructure (gradle, github, ...)
+- [ ] Other: 
+
+## Ensuring quality
```

Comment:
This section is kind of strange. 

Some of these things are hard to check ("runtime performance"). On some other we can discuss for weeks ("javadoc"). And other things like "I wrote a something in key-docs" are not reflected. 

I would such get rid of these points. They do not bring anyone further, or makes the review easier. 

I think this templating becomes to "German" and remembers me on the stupid ISO27001-driven forms back in my apprenticeship. 

Things that make this review easier is something like: "I needed to refactor the method class#method by adding a new parameter. Following classes are automatically updated w/o further intentions."

Reply:
I took the idea of a `[ ]` list from a reference you suggested earlier, @wadoon.
I am happy with it and without it. Could you edit it?